crates/livesplit-title-abbreviations/src/lib.rs

```diff
@@ -165,6 +165,12 @@ pub fn abbreviate(name: &str) -> Vec<Box<str>> {
     list.sort_unstable();
     list.dedup();
 
+    if let Some(idx) = list.iter().position(|x| name == x.as_ref()) {
+        list.remove(idx);
```

Comment:
rationale: we need to remove the full name from the list if it already exists, and at the same time i'd like to preserve the sorting

if speed is desired over sorting, this can be changed to a `Vec::swap()`

Reply:
Yeah `swap` almost definitely makes more sense as there is no guaranteed order in here other than the last element.